From 43aedaccb528b1f551838ec553c84a5ba47a6bdc Mon Sep 17 00:00:00 2001
From: Pradeep Sriram <pradeep.sriram@amlogic.com>
Date: Thu, 30 Apr 2020 13:00:49 -0700
Subject: [PATCH 3/9] RDK: migrate uboot code base to trunk [3/3]

PD#SWPL-24384

Problem:
 Build break due to werror

Solution:
  remove werror

Verify:
  Local

Signed-off-by: Pradeep Sriram <pradeep.sriram@amlogic.com>
Change-Id: Ib6f7ab8db107240e9263411d97dd38ac9c53d2f1
---
 arch/arm/cpu/armv8/g12a/firmware/scp_task/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/cpu/armv8/g12a/firmware/scp_task/Makefile b/arch/arm/cpu/armv8/g12a/firmware/scp_task/Makefile
index 7917e2042e..873b8e6600 100644
--- a/arch/arm/cpu/armv8/g12a/firmware/scp_task/Makefile
+++ b/arch/arm/cpu/armv8/g12a/firmware/scp_task/Makefile
@@ -16,7 +16,7 @@ OBJDUMP=$(CROSS_COMPILE)objdump
 
 ARM_ARCH=armv7-m
 ARM_CPU=cortex-m3
-CFLAGS_WARN=-Wall -Werror -Wundef -Wstrict-prototypes -Wno-trigraphs      \
+CFLAGS_WARN=-Wall -Wundef -Wstrict-prototypes -Wno-trigraphs      \
             -fno-strict-aliasing -fno-common -fomit-frame-pointer         \
             -Werror-implicit-function-declaration -Wno-format-security    \
             -fno-delete-null-pointer-checks -Wdeclaration-after-statement \
-- 
2.17.1

